console/client/src/providers/tabs-provider.tsx

```diff
@@ -1,33 +1,34 @@
-import { PropsWithChildren, createContext, useState } from 'react'
-import { Module, Verb } from '../protos/xyz/block/ftl/v1/console/console_pb'
+import React from 'react'
 
-export enum TabType {
-  Timeline = 'timeline',
-  Verb = 'verb',
-}
+export const TabType  = {
+  Timeline: 'timeline',
+  Verb: 'verb',
+} as const
 
 export type Tab = {
   id: string;
   label: string;
-  type: TabType;
-  module?: Module | null;
-  verb?: Verb | null;
+  type: typeof TabType[keyof typeof TabType];
 }
 
+export const TabSearchParams = {
+  active: 'active-tab',
+} as const
```

Comment:
I think this should have a structure more like this to make it more generic:

```ts
export const TabSearchParams = {
  type: 'tab-type',
  id: 'tab-id',
} as const
```

Then we could pass generic tab info in the query params instead of `active-tab=echo.echo` for example. If we update this structure we could support opening different tab types as well. Some examples might be:

```
tab-type=verb&tab-id=echo.echo
tab-type=request&tab-id=<RequestId>
etc.
```